src/App/_tests_/Redux/Reducers/authors.test.js

```diff
@@ -0,0 +1,154 @@
+import authorsReducer from '../../../../Redux/Reducers/authors';
+import actions from '../../../../Redux/Actions';
+
+describe('TERMS Reducer', () => {
+  it('FETCH_AUTHORS_PROFILE_SUCCESS reducer', () => {
+    const initialState = { fetchAuthors: true };
+    expect(authorsReducer(initialState, {
+      type: actions.FETCH_AUTHORS_PROFILE_SUCCESS,
+      payload: {
+        authors: [
+          {
+            id: 2,
+            username: 'BurindiAlain2',
+            email: 'alain2@gmail.com',
+            bio: null,
+            profileImage: null,
+            Articles: [{ id: 5 }, { id: 6 }],
+            followers: [
+              { follower: 'Burindi', profileImage: null },
+              { follower: 'BurindiAlain3', profileImage: null },
+              { follower: 'BurindiAlain44', profileImage: null }
+            ]
+          },
+          {
+            id: 42,
+            username: 'nkpremices',
+            email: 'premices.tuvere@gmail.com',
+            bio: 'Here I am',
+            profileImage:
+                'https://res.cloudinary.com/al-tech/image/upload/v1567175923/m5bfsomeowj7jsq6b2cs.jpg',
+            Articles: [{ id: 7 }],
+            followers: [{}]
+          },
+          {
+            id: 1,
+            username: 'Burindi',
+            email: 'alain1@gmail.com',
+            bio: 'I like dancing',
+            profileImage: 'defaultAvatar.jpg',
+            Articles: [{ id: 2 }, { id: 3 }, { id: 4 }, { id: 1 }],
+            followers: [
+              { follower: 'BurindiAlain2', profileImage: 'defaultAvatar.jpg' },
+              { follower: 'nkpremices', profileImage: 'defaultAvatar.jpg' }
+            ]
+          }
+        ],
+        follow: false,
+        paginationDetails: {
+          pages: 2,
+          currentPage: 1,
+          pageSize: 10,
+          count: 13
+        },
+        unfollow: false
+      }
+    })).toEqual({
+      fetchAuthors: true,
+      authors: [
+        {
```

Comment:
What do you think if you put this array in a constant then you expect that constant? to avoid duplication